### DISCUSSION TITLE
Fireman Switch: Only For Spa?

### DISCUSSION THREAD

**Discussion by bruno-monk** [tagyoureit/nodejs-poolController#761] [Nixie] [answered]
Hi there. Nixie 7.7.0 (Shared Body) with Sequent automation and relay HATs.
I just wired one of the relays (NO and COM) to old Laars EC400 heater.
Trouble is, the heater comes on for both pool and spa. Actually it comes on anytime the Intelliflo comes on: I'd like it for only the spa.
Pool set point is 0, Heat Mode is OFF.
Heater is set to "Spa."

Any advice to have the heater only active for SPA?

**Comment by rstrouse** [comment]
No the heater relay should be on its own.  That connects to the fireman's switch and will turn on the heater whether for the spa or pool.

**Reply by bruno-monk** [reply]
If one would like the heater only when the spa is isolated (through the valves controlled by the Nixie), is that possible? Basically I'd like to heat only when using the spa and not heat the entire pool. Thanks!

**Reply by tagyoureit** [reply]
The heater should only come on when the body and set point combo call for heat.  Something isn't right in your config or possibly wiring.  

Under the heater config is where you should associate the heater.  Do you have the relay defined as a circuit anywhere else?

**Reply by rstrouse** [reply, marked as answer]
Also in the heater config you can tell the controller which bodies the heater is valid for.  It will then only provide its options to that body.

**Reply by bruno-monk** [reply]
A reinstall of NPC took care of it: heater config was set appropriately but expected functionality was not happening.
